INTEGRATION.md

```diff
@@ -35,6 +35,39 @@ Integration here has 2 different meanings.
 
 2. One way to handle the protocols is to use the [HTTP gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/). This document describes how to determine which gateway to use.
 
+### $CONFIG folder, $GWFILE and $LAGACY_GWFILE
+
+Throughout this spec you might see `$CONFIG/ipfs/...`. `$CONFIG` is a mere abbreviation for the config folder, the variable doesn't exist and only serves to make this spec more readable. This section defines where `$CONFIG` should point to in a cross-platform manner.
+
+As a reference, [this](https://github.com/cjbassi/platform-dirs-rs#path-list) list of configuration folders is used.
+
+| | with variables | full paths |
+| -------- | -------- | -------- |
+| Windows     | %APPDATA%/ipfs     | C:/Users/%USERNAME%/AppData/Roaming/ipfs     |
+| macOS     | $XDG_CONFIG_HOME/ipfs     | ~/Library/Application Support/ipfs     |
+| Linux     | $XDG_CONFIG_HOME/ipfs     | ~/.config/ipfs     |
+
+In the same manner as $CONFIG, $GATEWAY and $LEGACY_GATEWAY are names for this spec only to simplify reading.
+
+`$GWFILE` = `$CONFIG/gateway`
+`$LEGACY_GWFILE` = `~/.ipfs/gateway`
```

Comment:
IPIP is defining a location of a single `gateways` file, keep it simple:

```suggestion
| | with variables | default
| -------- | -------- | -------- |
| Windows     | %APPDATA%/ipfs/gateways     | C:/Users/%USERNAME%/AppData/Roaming/ipfs/gateways     |
| macOS     | $XDG_CONFIG_HOME/ipfs/gateways     | ~/Library/Application Support/ipfs/gateways     |
| Linux (per user)     | $XDG_CONFIG_HOME/ipfs/gateways     | ~/.config/ipfs/gateways     |
| Linux (global)   | /etc/ipfs/gateways     | /etc/ipfs/gateways |
```

Note, I've added `/etc/ipfs/gateways` to account for use cases where one wants to provide config to all users.